support/troubleshooting/troubleshooting-upgrades.adoc

```diff
@@ -0,0 +1,51 @@
+[id="troubleshooting-upgrades"]
+= Troubleshooting upgrades
```

Comment:
This assembly includes the various new and existing modules you want to help users with troubleshooting upgrades.